mobile/src/people/employees-list.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { FlatList, Text, View, StyleSheet, ListRenderItemInfo } from 'react-native';
+import { connect } from 'react-redux';
+
+import { Department } from '../reducers/organization/department.model';
+import { Employee } from '../reducers/organization/employee.model';
+import { EmployeesStore, EmployeeMap, EmployeeIdsGroupMap } from '../reducers/organization/employees.reducer';
+import { AppState } from '../reducers/app.reducer';
+import { EmployeesListItem } from './employees-list-item';
+import { employeesListStyles as styles } from './styles';
+import { StyledText } from '../override/styled-text';
+
+export interface EmployeesListProps {
+    employees: Employee[];
+}
+
+export class EmployeesList extends React.Component<EmployeesListProps> {
+    public render() {
+        const employees = this.props.employees.sort((a, b) => {
+            if (a.name < b.name) {
```

Comment:
? why does it return 0 if a.name > b.name?

Reply:
i guess you forgot .name in second clause. Also, please put it a separate file / function called employeeComparer(first, second).

Reply:
ok